libcxx/include/__vector/vector_bool.h

```diff
@@ -398,6 +398,8 @@ class _LIBCPP_TEMPLATE_VIS vector<bool, _Allocator> {
     __guard.__complete();
```

Comment:
You should also add a release note for this nice improvement!

Reply:
Note that this is actually fixing a regression and not really improving performance.